autorest/codegen/models/list_schema.py

```diff
@@ -62,3 +63,8 @@ def from_yaml(cls, yaml_data: Dict[str, str], **kwargs) -> "SequenceType":
             min_items=yaml_data.get('minItems'),
             unique_items=yaml_data.get('uniqueItems'),
         )
+
+    def imports(self) -> FileImport:
+        file_import = FileImport()
```

Comment:
You probably wants to merge from your internal type, in case this schema would need a specific import too